src/views/feed/ActivityFeed/components/GroupMessageItem.vue

```diff
@@ -7,7 +7,10 @@
       :avatar="avatar"
     >
       <template>
-          {{ notification.target2_name + ' ' }} {{ isInvite ? $t('activity-feed:invite') : $t('activity-feed:broadcast') + ' ' }}
+          <router-link :to="`/players/` + message.sender">
+            {{ notification.target2_name + ' ' }} 
+          </router-link>
+          {{ isInvite ? $t('activity-feed:invite') : $t('activity-feed:broadcast') + ' ' }}
```

Comment:
```suggestion
          <router-link :to="`/players/` + message.sender">
            {{ notification.target2_name }} 
          </router-link>
          {{ ' ' + isInvite ? $t('activity-feed:invite') : $t('activity-feed:broadcast') + ' ' }}
```

Reply:
Done, but slightly needed to change to make the ternary operator work as intended,